docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -7,7 +7,7 @@ tags: ["themes", "getting-started"]
 
 > _Updated July 9, 2019 to reflect using the `gatsby-plugin-mdx` package instead of the (now deprecated) gatsby-mdx package._
 
-### What is a Gatsby theme?
+## What is a Gatsby theme?
 
```

Comment:
Only the first heading here looks wrong to me, so I've restored the others. The linter seems to get confused when one heading is incorrect but subsequent headings aren't.

Reply:
Thanks for this and for your review! Some cases were really difficult to judge actually.